apps/dashboard/src/lib/utils/translations/en.json

```diff
@@ -458,15 +474,15 @@
 
   "course": {
     "navItems": {
-      "1": "News Feed",
-      "2": "Lessons",
-      "3": "Attendance",
-      "4": "Submissions",
-      "5": "Marks",
-      "6": "People",
-      "7": "Certificates",
-      "8": "Landing Page",
-      "9": "Settings"
+      "nav_1": "News Feed",
+      "nav_2": "Lessons",
+      "nav_3": "Attendance",
+      "nav_4": "Submissions",
+      "nav_5": "Marks",
+      "nav_6": "People",
+      "nav_7": "Certificates",
+      "nav_8": "Landing Page",
```

Comment:
@tunny17 Okay let me repeat my self here again, I remember saying this when you worked on the teachers landing page.

## USING NUMBERS IN A VARIABLE NAME IS BAD

Even if it is in a JSON or constant or variable name or function name.

Using numbers mean that you want to preserve a certain order and when the other change you need to update all variables. In this case if we add a new `nav` and we want it on number 5, then you need to update all the variables after 5 and every where that name is used in the code. This is an unnecessary requirement.

Instead use the name that matches what you want to achieve, like instead of `nav_1` it should be `nav_news_feed` or instead of `nav_2` say `nav_lessons`.

In addition to numbers causing bugs, a variable with numbers can also be confusing cause you don't know what it represents. Take for instance `nav_1` and `nav_news_feed` which of them means something to you? to me the later does because it tells me that this is a key for translation news feed in the navigation.

Reply:
understood